core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryRestApplication.java

```diff
@@ -89,12 +89,12 @@ public void setupResources(Configurable<?> config, SchemaRegistryConfig schemaRe
             SchemaRegistryResourceExtension.class);
 
     config.register(RootResource.class);
-    config.register(new ConfigResource(schemaRegistry));
-    config.register(new SubjectsResource(schemaRegistry));
+    config.register(new ConfigResource(schemaRegistry, schemaRegistryConfig));
```

Comment:
Perhaps instead add a `config()` method to  either `SchemaRegistry` or `KafkaSchemaRegistry`.  Then you don't have to pass `schemaRegistryConfig` in addition to `schemaRegistry` everywhere.